jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/BannersTests.java

```diff
@@ -3,89 +3,160 @@
-            eventsBanner.buttons().getButtonWithText(CONNECTION_SETTINGS.toString()).is().clickable();
-        }
-
-        @Test
-        public void actionsBannerTests() {
-            actionsBanner.show();
-            actionsBanner.is().displayed().and().has().text("No Internet connection");
-            actionsBanner.buttons().is().displayed().and().has().size(2);
-            actionsBanner.buttons().getButtonWithText(DISMISS.toString()).has().text(DISMISS.toString());
-            actionsBanner.buttons().getButtonWithText(DISMISS.toString()).is().clickable();
-            actionsBanner.buttons().getButtonWithText(RETRY.toString()).has().text(RETRY.toString());
-            actionsBanner.buttons().getButtonWithText(RETRY.toString()).is().clickable();
-            actionsBanner.buttons().getButtonWithText(DISMISS.toString()).click();
-            waitCondition(() -> actionsBanner.bannerContent().isNotVisible());
-            actionsBanner.bannerContent().is().notVisible();
-            actionsBanner.bannerActions().is().notVisible();
-        }
-
-        @Test
-        public void iconBannerTests() {
-            iconBanner.show();
-            iconBanner.is().displayed();
-            iconBanner.getIconFromContent().is().displayed();
-            iconBanner.buttons().getButtonByIndex(1).has().text(ACTION.toString());
-            iconBanner.buttons().getButtonByIndex(1).is().clickable();
-            iconBanner.buttons().getButtonByIndex(2).has().text(ACTION.toString());
-            iconBanner.buttons().getButtonByIndex(2).is().clickable();
-            jdiAssert(iconBanner.attrs().has("two-line"), Matchers.is(true));
-        }
-
-        @Test
-        public void twoLineBannerTests() {
-            twoLineBanner.show();
-            twoLineBanner.is().displayed();
-            twoLineBanner.buttons().getButtonWithText(DISMISS.toString()).has().text(DISMISS.toString());
-            twoLineBanner.buttons().getButtonWithText(DISMISS.toString()).is().clickable();
-            twoLineBanner.buttons().getButtonWithText(RETRY.toString()).has().text(RETRY.toString());
-            twoLineBanner.buttons().getButtonWithText(RETRY.toString()).is().clickable();
-        }
+	@BeforeClass
+	public void before() {
+		bannersPage.open();
+		waitCondition(() -> bannersPage.isOpened());
+		bannersPage.checkOpened();
+	}
+
+	@Test
+	public void singleBannerTests() {
+		singleBanner.show();
+		singleBanner.is().displayed();
+		singleBanner.toolbarTitle().has().text("My Document");
+		singleBanner.checkbox().isClickable();
+		singleBanner.has().text("We can't save your edits while you are in offline mode.");
+		singleBanner.checkbox().click();
+		singleBanner.has().css("position", "relative");
+	}
+
+	@Test
+	public void eventsBannerTests() {
+		eventsBanner.show();
+		eventsBanner.is().displayed();
+		eventsBanner.getIconFromContent().is().displayed();
```

Comment:
It has classes v-banner-icon and avatar. I think every 'icon' here should have the class v-banner-icon, but it might not be the avatar. So, I think that better to leave it as an icon